src/Service/ActivityPub/Page.php

```diff
@@ -44,7 +44,8 @@ public function __construct(
      */
     public function create(array $object): Entry
     {
-        $actor = $this->activityPubManager->findActorOrCreate($object['attributedTo']);
+        $actorUrl = $this->activityPubManager->getActorFromAttributedTo($object['attributedTo']);
```

Comment:
`getActorFromAttributedTo` can return null, meaning `$actorUrl` can be null. Which you might want to check, before trying to findActorOrCreate below.

Reply:
since `findActorOrCreate` is fine with getting `null` as the `url` and can return `null` itself I think this is not a problem